src/documents/to-do/DeveloperIntroductionToInfusionFramework.md

```diff
@@ -4,6 +4,595 @@ layout: default
+        message: "{helloWorld}.model.message"
+    },
+    modelListeners: {
+        "message": "{that}.sayHello"
+    },
+    invokers: {
+        // fluid.notImplemented is a function that specifically represents
+        // an unimplemented function that components deriving from this
+        // grade are intended to implement; this is called an "abstract
+        // method" in Java
+        sayHello: "fluid.notImplemented"
+    }
+});
+
+fluid.defaults("fluid.helloWorld.consoleHello", {
+    // This component has all of the characteristics of sayHello,
+    // except for its implementation in the invoker
+    gradeNames: ["fluid.helloWorld.sayHello"],
+    invokers: {
+        sayHello: {
+            "this": "console",
+            "method": "log",
+            // Here, "{that}" means the context of the current
+            // component configuration (consoleHello)
+            "args": ["{that}.model.message"]
+        },
+    }
+});
+
+fluid.defaults("fluid.helloWorld.displayHello", {
+    // This component has all of the characteristics of sayHello,
+    // except for its implementation in the invoker
+    gradeNames: ["fluid.helloWorld.sayHello"],
+    invokers: {
+        sayHello: {
+            "this": "{helloWorld}.dom.messageArea",
+            "method": "html",
+            "args": ["{that}.model.message"]
+        }
+    }
+});
+
+fluid.defaults("fluid.helloWorld.speakHello", {
+    // This component has all of the characteristics of sayHello,
+    // except for its implementation in the invoker
+    // We also "mix in" the fluid.textToSpeech component to give it
+    // the capability to access the browser's text to speech interface
+    gradeNames: ["fluid.textToSpeech", "fluid.helloWorld.sayHello"],
+    invokers: {
+        sayHello: {
+            // This style of Invoker allows us to refer to another
+            // existing invoker using IoC references - in this case the
+            // "queueSpeech" invoker that we have access to from mixing
+            // in the fluid.textToSpeech grade
+            "func": "{that}.queueSpeech",
+            "args": ["{that}.model.message"]
+        }
+    }
+});
+
+fluid.defaults("fluid.helloWorld", {
+    gradeNames: ["fluid.viewComponent"],
+    model: {
+        message: "Hello, World!"
+    },
+    selectors: {
+        messageArea: ".flc-messageArea"
+    },
+    listeners: {
+        "onCreate.announceSelf": {
+            "this": "console",
+            "method": "log",
+            "args": ["The helloWorld Component is ready"]
+        }
+    },
+    components: {
+        consoleHello: {
+            type: "fluid.helloWorld.consoleHello"
+        },
+        displayHello: {
+            type: "fluid.helloWorld.displayHello",
+        },
+        speakHello: {
+            type: "fluid.helloWorld.speakHello",
+        }
+    }
+});
+```
+
+### Summing Up
+
+This elaborate "Hello, World!" implementation has slowly introduced some of Infusion's core features. To summarize:
```

Comment:
Perhaps "moderately complex" instead of "elaborate" would be better here.  Also, "introduces" rather than "has slowly introduced" seems better.